### PR TITLE
Static html reporter test minor improvements

### PR DESCRIPTION
**Pull request by fviernau**
Some code style improvements, see individual commits.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/1157)

### CODE REVIEW

File: reporter/src/funTest/kotlin/reporters/StaticHtmlReporterTest.kt [hunk 1]
```diff
@@ -31,25 +31,27 @@ import java.io.File
 
 import javax.xml.transform.TransformerFactory
 
-class StaticHtmlReporterTest : WordSpec({
-    val ortResult = File("src/funTest/assets/static-html-reporter-test-input.yml")
-            .readValue<OrtResult>()
+class StaticHtmlReporterTest : WordSpec()
+{
```
Comment (mnonnenmacher) [inline reporter/src/funTest/kotlin/reporters/StaticHtmlReporterTest.kt]: This `{` should be in the line above.
Commit message should start with "StaticHtmlReporterTest: ...".

File: reporter/src/funTest/kotlin/reporters/StaticHtmlReporterTest.kt [hunk 2]
```diff
@@ -45,14 +45,19 @@ class StaticHtmlReporterTest : WordSpec()
...
-                StaticHtmlReporter().generateReport(ortResult, DefaultResolutionProvider(), CopyrightGarbage(), outputDir)
+                val actualReport = generateReport(ortResult)
 
-                val actualFile = outputDir.resolve("scan-report.html")
-                val expectedFile = File("src/funTest/assets/static-html-reporter-test-expected-output.html")
-
-                actualFile.readText() shouldBe expectedFile.readText()
+                val expectedReport = File("src/funTest/assets/static-html-reporter-test-expected-output.html")
+                        .readText()
+                actualReport shouldBe expectedReport
```
Comment (mnonnenmacher) [inline reporter/src/funTest/kotlin/reporters/StaticHtmlReporterTest.kt]: Maybe keep the empty line above this one?

File: reporter/src/funTest/kotlin/reporters/StaticHtmlReporterTest.kt (hunk 1 above)
Comment (fviernau) [reply reporter/src/funTest/kotlin/reporters/StaticHtmlReporterTest.kt]: done

File: reporter/src/funTest/kotlin/reporters/StaticHtmlReporterTest.kt (hunk 2 above)
Comment (fviernau) [reply reporter/src/funTest/kotlin/reporters/StaticHtmlReporterTest.kt]: done

**Pull request merged**